tt_eager/tt_dnn/op_library/sliding_window_op_infra/untilize_with_halo_config_generation_and_validation.py

```diff
@@ -185,20 +174,23 @@ def decompose_conv_into_shards_and_generate_tensor_metadata(
     tensor_metadata = []
     unpadded_input_shard_local_idx = 0
     core_id = 0
-    for padded_input_tensor_idx in range(len(pad_metadata)):
+    # logger.info(len(pad_metadata))
+    # logger.info(num_cores)
+    for padded_input_tensor_idx in range(len(pad_metadata)):  # //input_c ):
         pad_stick = pad_metadata[padded_input_tensor_idx]
-        if pad_stick:
-            tensor_metadata.append((pad_stick, 0, 0))
+        if pad_stick:  # == 0:
+            tensor_metadata.append((True, 0, 0))
```

Comment:
is `pad_stick` always true?

Reply:
yes, should i change it back?

Reply:
think you can move this statement to be common to both branches?

```
tensor_metadata.append((pad_stick, 0, 0))
if not pad_stick:
   ....
```

Reply:
nah guess it doesn't matter